### ISSUE TITLE
Script for easy Observium upgrade

### ISSUE THREAD

**Issue by varnav** [turnkeylinux/tracker#750]
I don't see any tool to upgrade Observium. But this can be easily done according to [manual](http://www.observium.org/docs/updating/).

It may be worth to add this script for easy upgrade:

```
rm -rf observium_old
cd /opt
mv observium observium_old
wget -Oobservium-community-latest.tar.gz http://www.observium.org/observium-community-latest.tar.gz
tar zxvf observium-community-latest.tar.gz
mv /opt/observium_old/rrd observium/
mv /opt/observium_old/*log* observium/
mv /opt/observium_old/config.php observium/
/opt/observium/discovery.php -u
/opt/observium/discovery.php -h all

```

**Comment by GringaoGit** [comment]
Hi,

Just propose a small adaptation of this script, it permits to keep the config file in the old folder, and also to have full path for mv and cp.
<pre>
rm -rf /opt/observium_old
mv /opt/observium /opt/observium_old
wget -O/opt/observium-community-latest.tar.gz http://www.observium.org/observium-community-latest.tar.gz
tar zxvf /opt/observium-community-latest.tar.gz -C /opt/
mv /opt/observium_old/rrd /opt/observium/
mv /opt/observium_old/*log* /opt/observium/
cp /opt/observium_old/config.php /opt/observium/
/opt/observium/discovery.php -u
/opt/observium/discovery.php -h all
</pre>

**Comment by GringaoGit** [comment]
Hi,

Couldn't reproduce the issue.. (with community edition) and your experimental script works like a charm.

**Comment by JedMeister** [comment]
Hi @GringaoGit - Thanks tons for your feedback. I'm not sure why I had errors, but great to hear that you didn't. We've just released v15.1, so I've bumped the target milestone for this issue and we'll look to include it in the next Observium release. Thanks again.

**Comment by JedMeister** [comment]
FYI, I have finally spent a bit of time on this and here's the latest script:

https://github.com/JedMeister/observium/blob/16.1-update/overlay/usr/local/bin/turnkey-update-observium

It depends on:

https://github.com/JedMeister/observium/blob/16.1-update/overlay/usr/local/bin/turnkey-get-observium-ver
&
`jq` (`apt update && apt install -y jq`)

Once the current PR has been merged, then those files should be found in https://github.com/turnkeylinux-apps/observium/tree/master/overlay/usr//local/bin (or within the v16.1+ appliance; `turnkey-update-observium`).
